Added information regarding cost_validator usage when queries define variables (#78)
mirumee/ariadne-website: docs/query-validators.md

)
```

### Using `cost_validator` when query requires variables

`cost_validator` require queries to be executable. This means that any variables defined in the query have to be provided to the validator too, otherwise, it will raise `GraphQLError` saying that `Argument was provided the variable which was not provided a runtime value`.

To resolve this one can use dynamic configuration of the `validation_rules`.

```python
type_defs = gql(
    """
    type Query {
        hello(id: Int!): String!
    }

>     """)

Reviewer: ```suggestion
    """
)
```

def get_validation_rules(context_value, document, data):
    return [cost_validator(maximum_cost=5, variables=data.get("variables"))]

> schema = make_executable_schema(type_defs)

Reviewer: ```suggestion
schema = make_executable_schema(type_defs)

```

graphql = Graphql(
    schema,

>     validation_rules=get_validation_rules

Reviewer: ```suggestion
    validation_rules=get_validation_rules,
```

)
```

> ### Exposing query variables to `cost_validator`

Reviewer: ```suggestion

### Exposing query variables to `cost_validator`
```

> ### Using `cost_validator` when query requires variables

Reviewer: ```suggestion
### Exposing query variables to `cost_validator`
```

```

### Exposing query variables to `cost_validator`

Cost validator will raise an error if query containing variables is made, but variable values are not made available to the validator. Use dynamic configuration to avoid this:

> 
> 

Reviewer: ```suggestion

```

```

### Exposing query variables to `cost_validator`

> `cost_validator` require queries to be executable. This means that any variables defined in the query have to be provided to the validator too, otherwise, it will raise `GraphQLError` saying that `Argument was provided the variable which was not provided a runtime value`.

Reviewer: ```suggestion
Cost validator will raise an error if query containing variables is made, but variable values are not made available to the validator. Use dynamic configuration to avoid this:
```

> To resolve this one can use dynamic configuration of the `validation_rules`.

Reviewer: ```suggestion
```
